### DISCUSSION TITLE
Agent for RHEL 6

### DISCUSSION THREAD

**Discussion by risinamit** [fluent/fluent-bit#5699] [Q&A] [answered]
Hi All,

Hope everybody is doing well.
I would like to know that " Was Fluent bit agent was developed for RHEL 6 or CentOS 6 in the past?"
Looking for this information, since there are high number of RHEL6 in my environment and want to use FB agent.

Regards,
Amit Kumar

**Comment by agup006** [comment, marked as answer]
Hi Fluent Bit is only community maintained for currently supported Operating Systems - similar to other major Open Source projects.

As a company (Calyptia) we have built on older OSes as part of our enterprise offering, including CentOS 6 / RHEL 6 which required quite a bit of modification compared to the normal build as well as testing.
